DataDefinitions/Ship.cs

```diff
@@ -411,8 +411,7 @@ public override string ToString()
 
         public string SpokenName(string defaultname = null)
         {
-            string ship = (defaultname ?? phoneticmodel) ?? Properties.Ship._ship;
-            string result = Properties.Ship.your + " " + ship;
+            string result;
```

Comment:
We can remove `result` altogether, and just `return` the computed values.

Reply:
Done.